_posts/2017-10-13-updates.markdown

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title: Hush, Keep it down now - Clones are scary!
+date: 2017-10-13  15:00:00
+categories: updates
+---
+
+Did you know you can copy / clone any learner group, along with all it's members? Well, you can!
+
+![group-clone-image](https://gallery.mailchimp.com/845c4ebabb5b5ae7a6372c715/images/863f7ebb-7e53-4cf2-9a40-58e72cf96742.png)
+
+If you are running v3.38.0 (or later), and have updated your frontend to the most recent version, it's easy. Just pick your group and select the "copy" icon, and you will be able to create a complete copy of everything in that group, either with or without the students populating it: all it's subgroups and sub-subgroups, all names and structures.
+
+If you select the option to copy the population of the group as well, then all the students will be copied into the new clone exactly as they are in the original.
+
+Questions? Comments? Feedback? Please let us know at [support@iliosproject.org](mailto:support@iliosproject.org) or in [https://team-ilios.slack.com/messages/help/](https://team-ilios.slack.com/messages/help/) (if you have not yet joined our slack channel, you can get started here: [https://ilios-slack.herokuapp.com/](https://ilios-slack.herokuapp.com/)).
+
+Please [download the most recent release](https://www.github.com/ilios/ilios/releases/latest){:target="_releases"} and update your frontend to take advantage of all the latest updates!
```

Comment:
This doesn't render correctly. I think remove the `{:target="_releases"}` or put it inside the `()`.